teuthology/task/internal/__init__.py

```diff
@@ -304,7 +304,7 @@ def fetch_binaries_for_coredumps(path, remote):
             dump_path = os.path.join(coredump_path, dump)
             dump_info = subprocess.Popen(['file', dump_path],
                                          stdout=subprocess.PIPE)
-            dump_out = dump_info.communicate()
+            dump_out, _ = dump_info.communicate()
```

Comment:
I think that if we know we're not going to use stderr, we should do:
`dump_out = dump_info.communicate()[0]`

Reply:
Done